cmd/orb-server/startcmd/start.go

```diff
@@ -314,10 +315,17 @@ func startOrbServices(parameters *orbParameters) error {
 		httpsig.NewSigner(httpsig.DefaultPostSignerConfig()),
 	)
 
+	monitoringSvc, err := monitoring.New(storeProviders.provider, vcStore, monitoring.WithHTTPClient(httpClient))
```

Comment:
I think this needs to be moved to a more generic package, i.e. not monitoring, since Sandra needs to also add proof handling here.

Reply:
I know, we can call the monitoring inside another `ProofHandler`. Or I can create a simple `middleware` chain.